dotCMS/src/main/java/com/dotmarketing/factories/MultiTreeAPIImpl.java

```diff
@@ -552,7 +561,8 @@ public void overridesMultitreesByPersonalization(final String pageId,
     public void overridesMultitreesByPersonalization(final String pageId,
                                                     final String personalization,
                                                     final List<MultiTree> multiTrees,
-                                                     final Optional<Long> languageIdOpt) throws DotDataException {
+                                                    final Optional<Long> languageIdOpt,
```

Comment:
Optional as params is not a good practice

Reply:
really it was like this before my changes, and to change I have to change it in several places